changelog.md

```diff
@@ -61,5 +61,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Display component definitions as a grid
 - Factorize the opening of git dialogs inside GitBranchMenu
 - Rename GitForm and GitConfigurationDialog to GitSettingsForm and GitSettingsDialog respectively
+- Rename global save button to upload to git button
```

Comment:
remove this because is not released yet :)

Reply:
ok